### PR TITLE
improved docs

### PR DESCRIPTION
**Pull request by irinascurtu**
Improved ServicePulse retry messages docs

### CODE REVIEW

File: servicepulse/intro-failed-message-retries.md [hunk 1]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
 There are two ways to deal with this scenario. If the issue in the sender can be fixed, there will be no new malformed messages coming. The ones already sent can be redirected to an endpoint that specifically addresses the malformed messages. [This sample](/samples/servicecontrol/fix-messages/) shows how this can be done using the ServiceControl redirects feature.
 
 If the sender cannot be fixed it is better to use a pipeline behavior to repair the messages at the receiver endpoint before they are passed to the handler. [This sample](/samples/pipeline/fix-messages-using-behavior/) demonstrates this scenario.
+
+
+## ServicePulse interface
+
+ServicePulse allows you to see failed messages and retry them. The UI groups these messages based on the endpoint that failed to process them. You can see the source, and the number of failed messages.
```
Comment (WilliamBZA) [inline servicepulse/intro-failed-message-retries.md:27]:
```suggestion
ServicePulse allows you to see failed messages and retry them. By default, the UI groups these messages based on the endpoint that failed to process them. You can see the source, and the number of failed messages.
```

**Review by WilliamBZA — APPROVED** [review]
Minor suggestion, not a dealbreaker

File: servicepulse/intro-failed-message-retries.md [hunk 2]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
 There are two ways to deal with this scenario. If the issue in the sender can be fixed, there will be no new malformed messages coming. The ones already sent can be redirected to an endpoint that specifically addresses the malformed messages. [This sample](/samples/servicecontrol/fix-messages/) shows how this can be done using the ServiceControl redirects feature.
 
 If the sender cannot be fixed it is better to use a pipeline behavior to repair the messages at the receiver endpoint before they are passed to the handler. [This sample](/samples/pipeline/fix-messages-using-behavior/) demonstrates this scenario.
+
+
+## ServicePulse interface
+
+ServicePulse allows you to see failed messages and retry them. By default, the UI groups these messages based on the endpoint that failed to process them. You can see the source, and the number of failed messages.
+
+![Failed Messages ](/servicepulse/images/retry/failed-mesage-groups-annotated.png)
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:29]:
```suggestion
![Failed Messages](./images/retry/failed-mesage-groups-annotated.png)
```

File: servicepulse/intro-failed-message-retries.md [hunk 3]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
...
+
+### Retrying failed messages
+
+The messages can be retried in different ways:
+ -
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:34]:
```suggestion

```

File: servicepulse/intro-failed-message-retries.md [hunk 4]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
...
+#### 1. One by one
+
+![Individual message retry](/servicepulse/images/retry/failed-mesage-groups-annotated.png)
+
+In this case, you will see the a notification showing that a retry has been been requested, and it is in progress.
+![Individual message retry in progress](/servicepulse/images/retry/one-retry-in-progress.png)
+
+#### 2. By the an entire group
+
+![Retry all](/servicepulse/images/retry/retry-all.png)
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:47]:
```suggestion
![Retry all](./images/retry/retry-all.png)
```

File: servicepulse/intro-failed-message-retries.md [hunk 5]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
...
+
+You will see a retry in progress notification for all the messages, as well as the number of messages failed decreased.
+
+![All retried in progress](/servicepulse/images/retry/all-retry-in-progress.png)
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:51]:
```suggestion
![All retried in progress](./images/retry/all-retry-in-progress.png)
```

File: servicepulse/intro-failed-message-retries.md [hunk 6]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
...
+ -
+ - [One by one](#the-messages-can-be-retried-in-different-ways-one-by-one)
+ - [By the an entire group](#the-messages-can-be-retried-in-different-ways-by-the-an-entire-group)
+
+#### 1. One by one
+
+![Individual message retry](/servicepulse/images/retry/failed-mesage-groups-annotated.png)
+
+In this case, you will see the a notification showing that a retry has been been requested, and it is in progress.
+![Individual message retry in progress](/servicepulse/images/retry/one-retry-in-progress.png)
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:43]:
```suggestion
![Individual message retry in progress](./images/retry/one-retry-in-progress.png)
```

File: servicepulse/intro-failed-message-retries.md [hunk 7]
```diff
@@ -20,3 +20,32 @@ If a message was retried and came back to ServiceControl as failed it might mean
...
+### Retrying failed messages
+
+The messages can be retried in different ways:
+ -
+ - [One by one](#the-messages-can-be-retried-in-different-ways-one-by-one)
+ - [By the an entire group](#the-messages-can-be-retried-in-different-ways-by-the-an-entire-group)
+
+#### 1. One by one
+
+![Individual message retry](/servicepulse/images/retry/failed-mesage-groups-annotated.png)
```
Comment (ramonsmits) [inline servicepulse/intro-failed-message-retries.md:40]:
```suggestion
![Individual message retry](./images/retry/failed-mesage-groups-annotated.png)
```

**Pull request merged**